src/organizations/components/OrgProfileTab/DeletePanel.tsx

```diff
@@ -156,11 +157,17 @@ const DeleteOrgButton: FC = () => {
 
   const handleSuspendOrg = () => {
     if (orgCanBeSuspended) {
-      dispatch(
-        showOverlay('suspend-org-in-paid-account', null, () =>
-          dispatch(dismissOverlay())
+      if (users.length > 1) {
```

Comment:
you might consider creating a variable that stores the value of this check and is named in a way that explains the concept, especially if the rules around it might are liable to change.

Reply:
👍